politeiawww/quiesce.go

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2020 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"encoding/hex"
+	"net/http"
+
+	pd "github.com/decred/politeia/politeiad/api/v1"
+	www2 "github.com/decred/politeia/politeiawww/api/www/v2"
+	"github.com/decred/politeia/util"
+)
+
+func (p *politeiawww) toggleQuiesce() (*www2.QuiesceReply, error) {
```

Comment:
This needs to follow the existing style for all routes.  If the route's name is `Quiesce` then it needs to have a `handleQuiesce` methods and a `processQuiesce` methods.